File: Day 089/392. Is Subsequence.py
class Solution(object):
    def isSubsequence(self, s, t):
        a = len(s)
        b = len(t)

        i = 0
        j = 0
        while i < a and j < b:
            if s[i] == t[j]:
                i += 1
            j += 1
        return i == a

File: Day 089/392. Is Subsequence.java
class Solution {
    public boolean isSubsequence(String s, String t) {
        int a = s.length();
        int b = t.length();
        int i=0;
        for (int j=0; i<a && j<b ;j++){
            if (s.charAt(i)==t.charAt(j)){
                i++;
            }
        }
        if (i==a){
            return true;
        }
        else{
            return false;
        }
    }
}